src/main/python/linuxband/mma/grooves.py

```diff
@@ -95,12 +99,13 @@ def __do_load_grooves(self, grooves_list, path):
         """
         Load grooves and recurse into subdirectories.
         """
-        for dirname, dirnames, filenames in os.walk(path): #@UnusedVariable            
+        for dirname, dirnames, filenames in os.walk(path):
```

Comment:
Same comment as before.

Reply:
Seems to me, that all three variable are used below, hence I think, the Unused directive is not necessary.